moki-daniel/buildforsdgcovid19estimator
# devMoki develop branch merged to master

**What does this PR do?**
Performs challenge one to four of the buildforsdg assessment, challenge one to three is javascript in the estimator.js file while challenge four is the estimatorUI folder with several files inside it.

**Description of tasks to be completed?**
-Done the javascript challenges in question one to three
-Create the user interface for covid19 estimator

**How should this be manually tested?**
Clone this repo and check on the files explained above, for the UI open the estimatorUI, then load index.html file using any latest browser then, click on the links in the navbar to open different sub pages of the web application.

**Any background image associated with the PR?
![continentestimationsubpage](https://user-images.githubusercontent.com/5906691/79116717-e8578080-7d91-11ea-96b6-3b7cee320eb8.png)
![countryestimationssubpage](https://user-images.githubusercontent.com/5906691/79116722-ea214400-7d91-11ea-9bdc-7a4264ab98ea.png)
![globalestimationshomepage](https://user-images.githubusercontent.com/5906691/79116726-eb527100-7d91-11ea-8ded-e7b43699d098.png)


**
